tools/tpm2_policysigned.c

```diff
@@ -130,6 +135,7 @@ static bool tpm2_tool_onstart(tpm2_options **opts) {
         { "ticket",         required_argument, NULL,  0  },
         { "timeout",        required_argument, NULL,  1  },
         { "raw-data",       required_argument, NULL,  2  },
+        { "cphash",         required_argument, NULL,  3  },
```

Comment:
Rename this option. In all the other tools we have **cphash** to retrieve the command parameter hash without executing the command itself.